bootloader/bootloadertest/bootloadertest.go

```diff
@@ -406,11 +406,17 @@ func (b *MockManagedAssetsBootloader) UpdateBootConfig(opts *bootloader.Options)
 	return b.UpdateErr
 }
 
-func (b *MockManagedAssetsBootloader) CommandLine(args []string) (string, error) {
+func (b *MockManagedAssetsBootloader) CommandLine(extra, mode string) (string, error) {
 	if b.CommandLineErr != nil {
 		return "", b.CommandLineErr
 	}
-	line := strings.Join(append([]string{b.StaticCommandLine}, args...), " ")
+	args := []string(nil)
+	for _, argSet := range []string{b.StaticCommandLine, extra, mode} {
```

Comment:
here we have `<static> <extra> <mode>`, but in the grub.cfg and in the grub.CommandLine impl we have `<mode> <static> <extra>`, shouldn't this be:

```suggestion
	for _, argSet := range []string{mode, b.StaticCommandLine, extra} {
```